lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -190,6 +199,15 @@ export default function SubmitGradeForm({ student }: SubmitGradeFormProps) {
           cancelLabel="Close"
         />
       )}
+      {!!submitGradeError && errorCode && (
+        <LaunchErrorDialog
+          busy={true}
+          onRetry={undefined}
```

Comment:
TS not happy about this but we might actually want a retry button here, we could start without one.